main.go

```diff
@@ -55,6 +55,7 @@ var (
 	flagAudience         = flag.String("audience", "", "override JWT audience value (aud)")
 	flagToken            = flag.String("token", "", "override OIDC token")
 	flagPrependUserAgent = flag.Bool("prepend-user-agent", true, "prepend a custom User-Agent header to requests")
+	flagServerUpMinutes  = flag.Int("server-up-minutes", 0, "Time in minutes the proxy server will run. 0 means unlimited")
```

Comment:
I think it would be better to allow this to be a time.Duration instead. Then we can use `time.ParseDuration`, and users can specify things like "1m30s" in a nice human-readable format.

Reply:
Good suggestion. Done